### ISSUE TITLE
Question about KVars and ValueElement

### ISSUE THREAD

**Issue by rxcolvin** [kwebio/kweb-core#416]
In the code below state that I'm creating in a KVar and assigning to an Input  Element value doesn't seem to get reflected in the browser either when assigned initially to the  inputElement. The  documentation suggests that the  link between a kvar and the element is bi-directional. 

Am I missing something obvious here?

Thx

```kotlin
import kweb.*

fun main() {
    Kweb(
        port = 2395
    ) {
        doc.body {
            val state = kvar("foo")
            div().new {

                val inputElement = input(
                    type = InputType.text,
                )
                inputElement.value = state
            }    

            div().new {
                with(button()) {
                    text("reset")
                    on.click {
                        state.value = "foo"
                    }
                }
            }

            div().new {
                with(button()) {
                    text("print")
                    on.click {
                        println("value=${state.value}")
                    }
                }

            }
        }
    }

}
```

**Comment by sanity** [comment]
It's possible that there is a problem with propagating Kvar changes from server to browser - typically people use this mechanism for browser->server transmission of the value.

However this should work, will investigate. Thank you for the bug report.

**Comment by sanity** [comment]
Fixed in `1.3.2` which I just released.

**Comment by rxcolvin** [comment]
Thanks.
